core-services/prow/02_config/janus-idp/_prowconfig.yaml

```diff
@@ -14,3 +14,15 @@ tide:
     - needs-rebase
     orgs:
     - janus-idp
+  - labels:
```

Comment:
Is this change intentional?

Reply:
Reverted with original change. Thanks for pointing it out